srcpkgs/eclib/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'eclib'
+pkgname=eclib
+version=20210503
+revision=1
+build_style=gnu-configure
+configure_args="--with-flint --with-boost"
+hostmakedepends="automake autoconf-archive libtool"
+makedepends="pari-devel ntl-devel flintlib-devel boost-devel"
+short_desc="Elliptic curve library, includes mwrank"
+maintainer="Gonzalo Tornaría <tornaria@cmat.edu.uy>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/JohnCremona/eclib"
+distfiles="https://github.com/JohnCremona/eclib/archive/refs/tags/${version}.tar.gz"
+checksum=b742e322f5aeffef7256c05a9b9f744a8f34cb52777c62764cce07a49b1d32fa
+
+pre_configure() {
+	./autogen.sh
```

Comment:
We usually try to use `autoreconf -fi`, does their script do more than that?

Reply:
```
aclocal
libtoolize --automake
automake --add-missing
autoconf
```

I'll try to check if `autoreconf -fi` is enough.

Reply:
`autoreconf -fi` is ok so I changed to that.